src/index.js

```diff
@@ -1,10 +1,22 @@
+// @flow
+
 import React from 'react';
 import ReactDOM from 'react-dom';
 import firebase from 'firebase';
+import {createStore} from 'redux';
+import {Provider} from 'react-redux';
+
+import reducer from './reducers';
+import {notAuthed, handleUserAuthed} from './actions/auth';
 
 import App from './App';
+
 import './index.css';
 
+const store = createStore(
+  reducer,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
```

Comment:
this mystery meat line copied directly out of https://github.com/zalmoxisus/redux-devtools-extension#11-basic-store